reference/docs-conceptual/dsc/quickstarts/website-quickstart.md

```diff
@@ -6,7 +6,7 @@ title:  Quickstart - Create a website with DSC
 
 > Applies To: Windows PowerShell 4.0, Windows PowerShell 5.0
```

Comment:
Let's move this "Applies to" line after the H1 and before the first paragraph.

Reply:
Done